src/elements/common/header/Logo.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import IconLogo from '../../../icons/general/IconLogo';
+import messages from '../messages';
+import './Logo.scss';
+
+export interface LogoProps {
+    isSmall?: boolean;
```

Comment:
the `isSmall` prop is not used in this component. Please remove it in this file and in the component that passes this prop.